### PR TITLE
Add if defs for Xbox explicit code

### PR DESCRIPTION
**Pull request by toddbellMSFT**
Customers reported this code wasn't working on a platform that Xbox code won't be able to run in.

### CODE REVIEW

File: source/code/include/playfab/PlayFabIXHR2HttpPlugin.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#if defined(PLAYFAB_PLATFORM_XBOX)
 #pragma once
```
Comment (Vicodex) [inline source/code/include/playfab/PlayFabIXHR2HttpPlugin.h]: You may move this pragma higher, out of the protection block

**Comment by Vicodex** [comment]
> #pragma once

You may move pragma out of the block, for consistency

---
Refers to: source/code/include/playfab/PlayFabIXHR2HttpRequest.h:12 in 91650b9. [](commit_id = 91650b9daeaceeea237426d7e258f0a9202abe62, deletion_comment = False)

File: source/code/source/playfab/PlayFabIXHR2HttpPlugin.cpp [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #include <stdafx.h>
```
Comment (Vicodex) [inline source/code/source/playfab/PlayFabIXHR2HttpPlugin.cpp]: you may move stdafx include under the protection block

File: source/code/source/playfab/PlayFabIXHR2HttpRequest.cpp [hunk 3]
```diff
@@ -11,6 +11,8 @@
 
 #include <stdafx.h>
```
Comment (Vicodex) [inline source/code/source/playfab/PlayFabIXHR2HttpRequest.cpp]: Same here, you may move it under

File: source/code/include/playfab/PlayFabIXHR2HttpPlugin.h (hunk 1 above)
Comment (pgilmorepf) [reply source/code/include/playfab/PlayFabIXHR2HttpPlugin.h]: 👍

File: source/code/source/playfab/PlayFabIXHR2HttpPlugin.cpp (hunk 2 above)
Comment (pgilmorepf) [reply source/code/source/playfab/PlayFabIXHR2HttpPlugin.cpp]: I disagree Vicodex. In this case, you can't. It'll error if #include <stdafx.h> is not present, and the first code line, even if there's nothing else in the .cpp file.

File: source/code/source/playfab/PlayFabIXHR2HttpRequest.cpp (hunk 3 above)
Comment (pgilmorepf) [reply source/code/source/playfab/PlayFabIXHR2HttpRequest.cpp]: Repeat, I disagree, and it's likely a compile error.

**Pull request merged**